components/automate-ui/src/app/pages/project/details/project-details.component.spec.ts

```diff
@@ -145,11 +145,14 @@ describe('ProjectDetailsComponent', () => {
     });
 
     it('shows/hides sections when based on selection', () => {
-      component.onTabChange({ target: { value: 'details' } });
+      let tabName: ProjectTabName = 'details';
+      component.onTabChange({ target: { value: tabName } });
```

Comment:
🤔 This makes we wonder we can't give `onTabChange` a better type instead. Instead of `onTabChange(event)` we could go with
```typescript
onTabChange(event: { target: { value: ProjectTabName } })
```
or something like that (haven't tested it). I can see how its called with a more generic event, but since this is a) erased in transpilation, and b) only used this way, we could tighten the types here, I think.

Reply:
Right you are--that worked just fine. 👍